molo/core/migrations/0050_data_migration_promoted_articles.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_promote_start_date(apps, schema_editor):
```

Comment:
We just need to confirm that `F()` expressions are allowed in datamigrations. If not, you'll have to change `app.get_model()` and use `from molo.core.models import ArticlePage`